+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue Dec  6 06:37:08 2022 - INFO  - Start to run test.
+ test_execution dbus.socket
+ service=dbus.socket
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-06 06:37:08'
+ test_restart dbus.socket
+ service=dbus.socket
+ systemctl restart dbus.socket
Job failed. See "journalctl -xe" for details.
+ CHECK_RESULT 1 0 0 'dbus.socket restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='dbus.socket restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'dbus.socket restart failed'
+ LOG_ERROR 'dbus.socket restart failed'
+ message='dbus.socket restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'dbus.socket restart failed'
Tue Dec  6 06:37:09 2022 - ERROR - dbus.socket restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Tue Dec  6 06:37:10 2022 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status dbus.socket
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'dbus.socket restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='dbus.socket restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'dbus.socket restart failed'
+ LOG_ERROR 'dbus.socket restart failed'
+ message='dbus.socket restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'dbus.socket restart failed'
Tue Dec  6 06:37:17 2022 - ERROR - dbus.socket restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Tue Dec  6 06:37:17 2022 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop dbus.socket
+ CHECK_RESULT 0 0 0 'dbus.socket stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='dbus.socket stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status dbus.socket
+ grep 'Active: inactive'
     Active: inactive (dead)
+ CHECK_RESULT 0 0 0 'dbus.socket stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='dbus.socket stop failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl start dbus.socket
Job failed. See "journalctl -xe" for details.
+ CHECK_RESULT 1 0 0 'dbus.socket start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='dbus.socket start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'dbus.socket start failed'
+ LOG_ERROR 'dbus.socket start failed'
+ message='dbus.socket start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'dbus.socket start failed'
Tue Dec  6 06:37:25 2022 - ERROR - dbus.socket start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Tue Dec  6 06:37:25 2022 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ grep 'Active: active'
+ systemctl status dbus.socket
+ CHECK_RESULT 1 0 0 'dbus.socket start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='dbus.socket start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'dbus.socket start failed'
+ LOG_ERROR 'dbus.socket start failed'
+ message='dbus.socket start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'dbus.socket start failed'
Tue Dec  6 06:37:32 2022 - ERROR - dbus.socket start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Tue Dec  6 06:37:33 2022 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled dbus.socket
+ service=dbus.socket
++ systemctl is-enabled dbus.socket
+ state=disabled
+ '[' disabled == enabled ']'
+ '[' disabled == disabled ']'
++ systemctl enable dbus.socket
++ awk '{print $3}'
++ grep 'Created symlink'
+ symlink_file=/etc/systemd/system/sockets.target.wants/dbus.socket
+ find /etc/systemd/system/sockets.target.wants/dbus.socket
/etc/systemd/system/sockets.target.wants/dbus.socket
+ CHECK_RESULT 0 0 0 'dbus.socket enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='dbus.socket enable failed'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl disable dbus.socket
Removed /etc/systemd/system/sockets.target.wants/dbus.socket.
+ find /etc/systemd/system/sockets.target.wants/dbus.socket
find: ‘/etc/systemd/system/sockets.target.wants/dbus.socket’: No such file or directory
+ CHECK_RESULT 1 0 1 'dbus.socket disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='dbus.socket disable failed'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ journalctl --since '2022-12-06 06:37:08' -u dbus.socket
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
Dec 06 06:37:08 anolis-riscv systemd[1]: Failed to listen on dbus.socket - D-Bus System Message Bus Socket.
Dec 06 06:37:24 anolis-riscv systemd[1]: Failed to listen on dbus.socket - D-Bus System Message Bus Socket.
+ CHECK_RESULT 0 0 1 'There is an error message for the log of dbus.socket'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of dbus.socket'
+ '[' -z 0 ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of dbus.socket'
+ LOG_ERROR 'There is an error message for the log of dbus.socket'
+ message='There is an error message for the log of dbus.socket'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of dbus.socket'
Tue Dec  6 06:37:40 2022 - ERROR - There is an error message for the log of dbus.socket
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 28'
+ message='../common/common_lib.sh line 28'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 28'
Tue Dec  6 06:37:41 2022 - ERROR - ../common/common_lib.sh line 28
+ return 0
+ systemctl start dbus.socket
Job failed. See "journalctl -xe" for details.
+ systemctl reload dbus.socket
+ grep 'Job type reload is not applicable for unit dbus.socket.'
Failed to reload dbus.socket: Job type reload is not applicable for unit dbus.socket.
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status dbus.socket
+ grep 'Active: active'
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_socket_dbus.sh line 29'
+ message='oe_test_socket_dbus.sh line 29'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_socket_dbus.sh line 29'
Tue Dec  6 06:37:42 2022 - ERROR - oe_test_socket_dbus.sh line 29
+ return 0
+ LOG_INFO 'End of the test.'
+ message='End of the test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of the test.'
Tue Dec  6 06:37:43 2022 - INFO  - End of the test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Dec  6 06:37:44 2022 - ERROR - failed to execute the case.
+ exit 6
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_socket_dbus.sh post_test'
+ message='oe_test_socket_dbus.sh post_test'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'oe_test_socket_dbus.sh post_test'
Tue Dec  6 06:37:45 2022 - INFO  - oe_test_socket_dbus.sh post_test
